test/unittest/tests/test_prepare_mod_sort.cpp

```diff
@@ -118,9 +118,9 @@ init_key(WT_SESSION_IMPL *session, WT_ITEM *key, std::string key_str)
     WT_DECL_RET;
 
     ret = __wt_buf_init(session, key, key_str.size());
-    WT_ASSERT(session, ret == 0);
+    REQUIRE(ret == 0);
```

Comment:
Does it belong to this PR?

Reply:
A good question, and not necessarily. It was a mistake I picked up as I was looking through the code. I could move it into a seperate ticket/PR.

Reply:
I've rolled back this change, and will create a separate ticket for it.

Reply:
I've created https://jira.mongodb.org/browse/WT-12364 to make this change.